src/server/api/data.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import "../config";
+
+interface Coordinates {
+  latitude: number;
+  longitude: number;
+}
+
+const API_KEY = process.env.TICKET_MASTER_API_KEY;
+
+export const getCoordinates = async (postcode: any): Promise<Coordinates> => {
```

Comment:
Can you type `postcode` better? `any` should be your last resort in Typescript, see [this article](https://mariusschulz.com/blog/the-unknown-type-in-typescript). If you really don't know what type to expect, `unknown` is a better choice.

Reply:
updated code - https://github.com/builtbybea/bass-and-treble/pull/2/commits/aeba47b2aa93b1ef74dda4ffbc93f6a80fdae47a